src/main/java/com/attacktimer/AttackTimerMetronomeConfig.java

```diff
@@ -39,7 +41,7 @@ default boolean enableMetronome()
 	)
 	default boolean showTick()
 	{
-		return false;
```

Comment:
I'd prefer to keep default settings the same so that users don't have any surprises after this plugin gets updated.

Reply:
I was thinking about other examples on plugin hub, main feature of the plugin should be ON by default, so new users don't get confused when they install the plugin for the first time and ask themselves "Why after I install the plugin isn't showing the tick counter? Isn't this the plugin for that?"

Lemme know what you think, if you don't agree we can just set it to false, not a biggie

Reply:
That makes sense. Let's leave this one defaulted to true.